rsocket-routing-broker/src/main/java/io/rsocket/routing/broker/rsocket/ResolvingRSocket.java

```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package io.rsocket.loadbalance;
+package io.rsocket.routing.broker.rsocket;
```

Comment:
I think this and (the other `~DeferedResolution` classes that are duplicated should have Javadoc that indicates where they came from and that they are duplicated and should be kept in sync.